qutip/tests/test_random.py

```diff
@@ -71,13 +72,34 @@ def check_func_dims(func, args, kwargs, dims):
     # TODO: promote this out of test_random, as it's generically useful
     #       in writing tests.
     resdims = func(*args, **kwargs).dims
-    assert_(resdims == dims, "Checking {}; epected dimensions of {}, got {}.".format(func.__name__, dims, resdims))
+    assert_(resdims == dims, "Checking {}; expected dimensions of {}, got {}.".format(func.__name__, dims, resdims))
```

Comment:
You wouldn't have known this necessarily, but now we use `pytest`, the better way to write this test is just to write
```python
assert resdims == dims
```
(Note that's a bare `assert` keyword, rather than numpy's slightly odd `assert_`.)  Pytest will show us the values of all the function parameters (`func`, `args`, `kwargs`) and both sides of the `==` (`resdims` and `dims`) if the test fails, which means we get the best possible error message without actually needing to write one at all.

Reply:
Would you prefer I remove the error message `Checking {}; expected dimensions of {}, got {}.` ? 
Like you said, pytest will try to give best possible error message.

Reply:
I'm happy if you don't want to bother right now (since you were just matching the style of the rest of the code), but that's up to you.